promscale/installation/docker.md

```diff
@@ -0,0 +1,123 @@
+# Test Promscale using a pre-built container
+If you want to try out Promscale in a test environment before getting started,
+you can use a pre-built Docker container.
+
+<highlight type="important">
+Using Promscale from a Docker container is for local testing purposes only. Do
+not use this installation method in a production environment.
+</highlight>
+
+## Install Promscale with Docker
+Before you begin, you must have Docker installed on your local system. For
+packages and instructions, see the
+[Docker installation documentation][docker-install].
+
+<procedure>
+
+### Installing Promscale with Docker
+1.  Use Docker to create a network for Promscale and TimescaleDB:
+    ```bash
+    docker network create --driver bridge promscale-timescaledb
+    ```
+1.  Install TimescaleDB in a Docker container, and make sure the service is
+    running. For instructions, see
+    [the TimescaleDB Docker installation section][tsdb-docker]. Promscale stores
+    all metrics data scraped from Prometheus targets in this TimescaleDB service.
+1.  Run the Promscale Docker container. This example creates a TimescaleDB
+    instance called `timescaledb`, on a network named `promscale-timescale`. The
+    container runs in the background, and the container ID is shown after it is
+    created. It also port forwards to port `5432` on your local system:
+    ```bash
+    docker run --name timescaledb \
+        --network promscale-timescaledb \
+        -e POSTGRES_PASSWORD=<password> -d -p 5432:5432 \
+        timescaledev/timescaledb-ha:pg12-latest
+    ```
+
+</procedure>
+
+## Collect metrics with node_exporter
+When you have Promscale running in a container, you can begin collecting metrics. The `node_exporter` tool is a service that exposes hardware and operating systems metrics to Prometheus. For more information about `node_exporter`, see the [node_exporter developer documentation][gh-node-exporter].
+
+<procedure>
+
+### Collecting metrics with node_exporter
+1.  Run the `node_exporter` Docker container. This example creates
+    a `node_exporter` instance called `node_exporter`, port forwarding output to
+    port `9100`, and it runs on the `promscale-timescaledb` network you created
+    earlier:
+    ```bash
+    docker run --name node_exporter -d -p 9100:9100 \
+    --network promscale-timescaledb \
+    quay.io/prometheus/node-exporter
+    ```
+1.  Verify that system metrics are being exported, by opening
+    `http://localhost:9100/metrics` in your browser. This is the location that
+    Prometheus scrapes for metrics data.
+
+</procedure>
+
+## Install Prometheus
+When you are succecssdfully receiving metrics from your Promscale instalaltion, you can install Prometheus and configure it to garther the metrics data.
+
+<procedure>
+
+### Installing Prometheus
+
+1.  Create a Prometheus configuration file called `prometheus.yml`. You can use
```

Comment:
```suggestion
1.  Create a Prometheus configuration file `prometheus.yml`. You can use
```

Reply:
This content was removed so the problem doesn't exist anymore